src/source/reference/get-user.rst

```diff
@@ -3,13 +3,46 @@
 
 GET /user
 =========
-:ref:`get_user_v3` | :ref:`get_user_v1`
+:ref:`get_user_v4` | :ref:`get_user_v3` | :ref:`get_user_v2` | :ref:`get_user_v1`
+
+This URL returns information of the current user. The ``id`` field contains
+a unique user identifier. The ``options.feed`` field is ``true`` if the user
+is an XML feed customer. The ``options.vanityUrl`` field is ``true`` if the
+value of the :ref:`ad_links_displayUrl` field in the advertisement is shown
+in the search results. The ``hasAds`` field is ``false`` if the user has no ads at all (not even deleted).
+The ``locations`` field contains the user (or webshop) locations.
+The ``isAPIManaged`` field is ``true`` when there is (at least one) API partner
+using RW-scopes for this customer. API partners using RO-scopes only will not
```

Comment:
why don't we use the same lingo as the Scope section `api_rw` (also makes it clear that we don't treat console as API partner :) )

Reply:
good point, wll check that out.